src/btree/bt_split.c

```diff
@@ -340,9 +340,18 @@ __split_ref_move(WT_SESSION_IMPL *session, WT_PAGE *from_home,
 			return (ret);
 		}
 		addr->size = (uint8_t)unpack.size;
-		addr->type =
-		    unpack.raw == WT_CELL_ADDR_INT ? WT_ADDR_INT : WT_ADDR_LEAF;
-		ref->addr = addr;
```

Comment:
Oh, right -- yes, that was a typo, only meant to touch `addr->type`.  I'll revert that line...